packages/block-editor/src/components/inspector-controls/index.native.js

```diff
@@ -30,7 +39,9 @@ const FillWithSettingsButton = ( { children, ...props } ) => {
 					</BottomSheetConsumer>
 				}
 			</Fill>
-			{ React.Children.count( children ) > 0 && <BlockSettingsButton /> }
+			{ React.Children.toArray( children ).filter(
+				( child ) => child.props.dontCount !== true
```

Comment:
@antonis Thank you for adding the comment above on `dontCount`. To be honest, though I still had to reread that a few times to understand what the setting was meant to do. I wonder if we can rename it to something like `excludeFromSettings` or `excludeFromCount` something along those lines. WDYT?


```suggestion
				( child ) => child.props.excludeFromSettings !== true
```

Reply:
I agree with you @chipsnyder and made the change. I still find this implementation a bit dirty but I couldn't find a better way to achieve this :|